apps/router-e2e/__e2e__/server/app/api/[dynamic]+api.ts

```diff
@@ -1,6 +1,4 @@
-import { ExpoRequest, ExpoResponse } from 'expo-router/server';
-
-export function GET(req: ExpoRequest): ExpoResponse {
+export function GET(_req: Request, { dynamic }): Response {
   // curl -d "param1=value1&param2=value2" -X POST http://localhost:8082/data
-  return ExpoResponse.json({ hello: req.expoUrl.searchParams.get('dynamic') });
+  return new Response(JSON.stringify({ hello: dynamic }));
```

Comment:
Isn't Response.json on the type? https://developer.mozilla.org/en-US/docs/Web/API/Response/json_static

Reply:
Depending on the TypeScript version and used types this won't always be available, and we can't “force” it to be there, since we can't re-declare `var Response` 😅 
In our case it's missing because we're on an older TypeScript version

Reply:
Can we do something like this for `.json` as well? https://github.com/expo/expo/pull/27261/files#diff-4ab881a1080ac3bedcac4050238c0cb055661849b325b897c27ebf16bf3ee192R45-R62